GlobalSuppressions.cs

```diff
@@ -5,13 +5,25 @@
 // a specific target and scoped to a namespace, type, member, etc.
 
 //XForms
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid:Avoid async void", Justification = "<Pending>", Scope = "member", Target = "~M:XForms.AcquirePage.OnAcquireClicked(System.Object,System.EventArgs)")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid:Avoid async void", Justification = "<Pending>", Scope = "member", Target = "~M:XForms.AcquirePage.OnAcquireSilentlyClicked(System.Object,System.EventArgs)")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid:Avoid async void", Justification = "Reviewed", Scope = "member", Target = "~M:XForms.AcquirePage.OnAcquireClicked(System.Object,System.EventArgs)")]
```

Comment:
>Reviewed [](start = 144, length = 8)

imo, not a good justification. More like Top level event handlers calling async can be void if appropriate